src/client/containers/EmptyQuizResultPage/EmptyQuizResultPage.container.js

```diff
@@ -23,11 +24,16 @@ export const EmptyQuizResultPage = () => {
             <div className="sadrobo">
               <img src={sadrobot} alt="disappointed robo face" />
             </div>
-            <div className="sadrobo_text">
-              <p>You`ve skipped too many questions</p>
-            </div>
+            <p className="sadrobo_text">You`ve skipped too many questions</p>
           </div>
-          <Buttons label="Go Home" size="big" isMono={true} onClick={onClick} />
+          <Link to="/">
+            <Buttons
+              label="Go Home"
+              size="big"
+              isMono={true}
+              onClick={onClick}
+            />
+          </Link>
```

Comment:
This is not the correct way to write this. think of it as it ends up being HTML and you should not wrap an <a> tag around a <button> tag.

https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5
 
If you want to navigate to a page using a button this is the correct way to do so :).

https://reactrouter.com/web/api/Hooks/usehistory

Reply:
> This is not the correct way to write this. think of it as it ends up being HTML and you should not wrap an tag around a tag.
> 
> https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5
> 
> If you want to navigate to a page using a button this is the correct way to do so :).
> 
> https://reactrouter.com/web/api/Hooks/usehistory

Hi @bytewiz i did all the changes as you requested, Thank You Niels